src/main/java/com/blackducksoftware/integration/hub/alert/datasource/entity/global/GlobalSlackConfigEntity.java

```diff
@@ -36,36 +36,20 @@
 public class GlobalSlackConfigEntity extends DatabaseEntity {
     private static final long serialVersionUID = 4872590789715928839L;
 
-    @Column(name = "channel_name")
-    private String channelName;
-
-    @Column(name = "username")
-    private String username;
```

Comment:
I'm inclined to keep the username in the global config so that it is associated with the webhook.

E.g. "Hub Alert Bot" would be the username of the webhook being used.

Reply:
+1